src/TableCellHead.js

```diff
@@ -46,15 +46,13 @@ export const TableCellHead = ({
 /**
  * @typedef {Object} PropTypes
  * @static
- * @prop {string} [label]
  * @prop {string} [colSpan]
  * @prop {string} [rowSpan]
  * @prop {bool} [dense]
  * @prop {Node} [children]
  * @prop {string} [className]
  */
 TableCellHead.propTypes = {
-    label: propTypes.string,
```

Comment:
Possible breaking change: TableCellHead no longer accepts a label

Reply:
No, it was an unused prop so ESLint threw a warning about it.